packages/react/src/components/pickers/BasePicker.tsx

```diff
@@ -34,7 +34,7 @@ import { IPickerItemProps } from './PickerItem.types';
 import * as stylesImport from './BasePicker.scss';
 const legacyStyles: any = stylesImport;
 
-export interface IBasePickerState {
+export interface IBasePickerState<T> {
```

Comment:
Should we name this something more descriptive like `TItem`?

Reply:
Generally I'd agree, but in this case the specific generic `<T>` is what's used across all the BasePicker interfaces, so I kept it consistent since it's the same generic being passed in to all of them.